src/MyMap/MyHashMap.java

```diff
@@ -48,25 +57,26 @@ public void put(Object key, Object value) {
 
             } else if (capacity - size == 1) {
                 //TODO change name to something like normalizeSize
-                resize();
+                normalizeMap();
             }
 
+
             Object[] pairKeyValue = new Object[2];
```

Comment:
if you need this object only in state when there no such a key in table than you can create it in "if (GetRemoveHelper(key, Flag.GET) == null)"

Reply:
ok, done!